dump26/claims/text.py

```
@@ -123,38 +209,129 @@ def make_numbers_section(p31_list):
-    metadata += f"* Total items: {data['All_items']:,}\n"
-    metadata += f"* Items without P31: {data['items_no_P31']:,}\n"
-    metadata += f"* Items without claims: {data['items_0_claims']:,}\n"
-    metadata += f"* Items with 1 claim only: {data['items_1_claims']:,}\n"
-    metadata += f"* Total number of claims: {data['total_claims']:,}\n"
-    metadata += f"* Number of properties in the report: {data['len_all_props']:,}\n"
+    metadata += facts(data, Old)
+    # ---
+    Old_props = Old.get("properties", {})
     # ---
     final = time.time()
     delta = data.get("delta") or int(final - time_start)
```

Ruff (0.8.2)

Local variable `delta` is assigned to but never used

Remove assignment to unused variable `delta`

(F841)